custom_components/proxmoxve/__init__.py

```diff
@@ -379,6 +379,7 @@ def get_data_node(
 ) -> dict[str, Any]:
     """Get the node data in two API endpoints."""
     api_status = proxmox.nodes(node).status.get()
+    api_status.update(proxmox.nodes(node).storage('local-zfs').status.get())
```

Comment:
This will break the integration for users who have not named the storage `local-zfs`. So this needs to come from a configuration entry, where the user selects which storages he wants to include in the integration.

So it should be treated the same as QMEU/LXC in terms of handling errors and creating user fixes if that resource no longer exists on the server.